docs/README.md

```diff
@@ -121,7 +121,9 @@ The Media Communications Mesh (MCM) enables efficient, low-latency media transpo
 The program "media_proxy" and SDK library will be installed on system, after the "build.sh" script run successfully. To run "Media Proxy" execute below command:
 
 ```bash
-$ media_proxy
+media_proxy
+```
+```
```

Comment:
I would add here (before INFO: TCP Server..." the line. sth like:
The result should look like this:

Reply:
Fixed.